src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -483,6 +497,53 @@ def testManyJobs(self):
             # Make sure we have the right number of jobs
             self.assertEquals(len(allJobs), 3001)
 
+        def testImportFileVaryingFileSizes(self):
+            self._runTestMethodWithParams(self._importFileFromFileOfSize, [0, 1, 2**20, 2**20*5])
```

Comment:
I would make the larger sizes variables and/or include comments so it's more obvious why these sizes are being chosen